app/src/main/res/layout/activity_login.xml

```diff
@@ -17,13 +15,20 @@
     app:layout_behavior="@string/appbar_scrolling_view_behavior"
     >
 
-    <ImageView
+    <LinearLayout
       android:layout_width="match_parent"
       android:layout_height="0dp"
       android:layout_weight="1"
+      android:orientation="vertical"
+      android:gravity="center">
```

Comment:
Not following the code style, the `>` closing the tag should be on the next line.